### DISCUSSION TITLE
ltsp intird script bash

### DISCUSSION THREAD

**Discussion by RichardBachelier** [ltsp/ltsp#896] [General]
Hello,

"I want to use the 'ltsp initrd' command to navigate and modify the ltsp image scripts without recompiling it. I have created a directory called 'myscripts' in /etc/ltsp/initrd/. It contains several bash scripts (*.sh). When I run the 'ltsp initrd' command, I encounter a runtime error on certain scripts. For example:

/tmp/tmp.6IT2qJKrvo/etc/ltsp/initrd/bin/wk4-logsdnsfilter.sh: 13: Syntax error: "(" unexpected
Syntax error in initrd script: /tmp/tmp.6IT2qJKrvo/etc/ltsp/initrd/bin/wk4-logsdnsfilter.sh"

How can I proceed? Thank you for your assistance.

Best regards

Richard

**Comment by alkisg** [comment]
This means that your script has a syntax error, just fix your script.
Try `sh -n /etc/ltsp/initrd/bin/wk4-logsdnsfilter.sh` yourself to pinpoint the issue.
